angular-workspace/projects/ni/nimble-angular/src/directives/table/nimble-table.directive.ts

```diff
@@ -7,6 +7,8 @@ import type { Observable, Subscription } from 'rxjs';
 export type { Table };
 export type { TableActionMenuToggleEventDetail, TableRowSelectionEventDetail };
 export { TableRecord, TableFieldName, TableFieldValue, TableValidity, TableRowSelectionMode };
+// eslint-disable-next-line no-restricted-imports
+export * from '@ni/nimble-components/dist/esm/table/testing/table.pageobject';
```

Comment:
I don't think it makes sense to export from the directive. The idea would be to have separate `@ni/nimble-angular/table` and `@ni/nimble-angular/table/testing` entry points.

Hmm actually I don't think there is any reason to not try that with the table now?
What do you think about moving the table to that layout?

https://github.com/ng-packagr/ng-packagr/blob/main/docs/secondary-entrypoints.md